styles/components/App.less

```diff
@@ -30,12 +29,12 @@
   margin-left: auto;
   margin-right: auto;
   margin-bottom: 20px;
-  min-height: 100px;
-  max-height: 420px;
+  min-height: 470px;
```

Comment:
Again I think there are undesirable side-effects to these changes - the footer is partially or completely cut off after login on some screens, for example.